README.md

```diff
@@ -39,11 +39,21 @@ part of package.json
 ```
 
 ## Pre-requisites
-If you're on OS X/Linux and want to build for Windows, you need Wine installed. Wine is required in order to set the correct icon for the exe.
+If you're on OS X/Linux and want to build for Windows, you need Wine installed. Wine is required in order to set the correct icon for the exe. 
 
+You will also need the nullsoft scriptable install system for all platforms.
+
+On osx via brew
 ```
 $ brew install wine makensis
 ```
+On linux via apt-get
```

Comment:
Ok. I'll fix it in the other PR. This was branched off of justin/win-32 so it has some duplicate commits. When that other one gets merged in then they should drop off this PR.